Define HPKE-dependent pieces correctly (#168)
ietf-wg-ohai/oblivious-http: draft-ietf-ohai-ohttp.md

~~~
Encapsulated Response {

>   Nonce (max(Nn, Nk)),

Reviewer: ```suggestion
  Nonce (8 * max(Nn, Nk)),
```

Author: These are already defined in bits, so we don't need this change.

Reviewer: Nn and Nk are bytes, so this would yield the wrong number of bits here, wouldn't it?

Author: Sorry, you are right.  I should double check before making bad claims.

  AEAD-Protected Response (..),
}
~~~
{: #fig-enc-response title="Encapsulated Response"}

The Nn and Nk values correspond to parameters of the AEAD used in HPKE, which is
defined in {{Section 7.3 of !HPKE}}.  Nn and Nk refer to the size of the AEAD

> nonce and key respectively, in bits.  The nonce length is set to the larger of

Reviewer: ```suggestion
nonce and key respectively, in bytes.  The Encapsulated Response nonce length is set to the larger of
```

> these two lengths.

Reviewer: ```suggestion
these two lengths, i.e., max(Nn, Nk).
```

> nonce and key respectively, in bytes.  The Encapsulated Response nonce length is set to the larger of
> these two lengths, i.e., max(Nn, Nk).

Author: ```suggestion
nonce and key respectively, in bytes.  The Encapsulated Response nonce length is
set to the larger of these two lengths, i.e., max(Nn, Nk).
```
